client/src/styles/global.scss

```diff
@@ -1,2 +1,24 @@
-@import "~@trussworks/react-uswds/lib/uswds.css";
 @import "~@trussworks/react-uswds/lib/index.css";
```

Comment:
Ah, actually just noticed this, but this should be in a css module scoped to the header - you can copy what `layout.module.scss` has done here

Reply:
we discussed offline - will do this once we have a fix for gatsby-plugin-scss-typescript we can use modules directly, will keep in global for now